src/registry_schemas/schemas/alteration.json

```diff
@@ -58,10 +58,6 @@
                     "type": "string",
                     "title": "The Identifier for rules file in file server"
                 },
-                "rulesFileName": {
```

Comment:
Not needed, they are present for the draft, but I don't believe are used in the filing itself